fastlane/Fastfile

```diff
@@ -51,6 +51,7 @@ lane :automatic_bump do |options|
     github_rate_limit: options[:github_rate_limit]
   )
   options[:next_version] = next_version
+  UI.user_error!('Skipping automatic bump since the next version doesn\'t include public facing changes') if type_of_bump == :skip
```

Comment:
🤔 public facing changes here might feel like it means a minor release. But this is for anything that isn't CI, right? 
Maybe "non-infrastructure changes"?